torch/nn/quantized/dynamic/modules/linear.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ....modules.module import Module
+from ....modules.linear import Linear as NNLinear
+
+class Linear(NNLinear):
+    r"""
+    A dynamic quantized linear module with quantized tensor as inputs
+    and outputs.
+    We adopt the same interface as `torch.nn.Linear`, please see
+    https://pytorch.org/docs/stable/nn.html#torch.nn.Linear for documentation.
+
+    Similar to `torch.nn.Linear`, attributes will be randomly initialized at
+        module creation time and will be overwritten later
+
+    Attributes:
+        weight: the non-learnable quantized weights of the
+                module which are of shape :math:`(\text{out\_features}, \text{in\_features})`.
+        bias:   the non-learnable bias of the module of shape :math:`(\text{out\_features})`.
+                If :attr:`bias` is ``True``, the values are initialized to zero.
+        scale: `scale` parameter of weight Quantized Tensor, type: double
+        zero_point: `zero_point` parameter for weight Quantized Tensor, type: long
+
+    Examples::
+
+        >>> m = nn.quantized.Linear(20, 30)
+        >>> input = torch.randn(128, 20)
+        >>> output = m(input)
+        >>> print(output.size())
+        torch.Size([128, 30])
+    """
+
+    __constants__ = ['bias', 'in_features', 'out_features', 'scale', 'zero_point']
+
+    def __init__(self, in_features, out_features, bias=True):
```

Comment:
seems like you ignore bias=False/True

Reply:
Right: Will add the handler for bias = None case.

Reply:
Updated.